### ISSUE TITLE
Support Flutter v1.20.1

### ISSUE THREAD

**Issue by MarsGoatz** [jpnurmi/cupertino_list_tile#1]
Thank you for this package. There are some breaking changes when upgrading Flutter to v1.20.1, so creating an issue.

```
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:168:17: Error: Type 'ActionFactory' not found.
  Map<LocalKey, ActionFactory> _actionMap;                              
                ^^^^^^^^^^^^^                                           
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:168:17: Error: 'ActionFactory' isn't a type.
  Map<LocalKey, ActionFactory> _actionMap;                              
                ^^^^^^^^^^^^^                                           
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:177:22: Error: Getter not found: 'key'.
      ActivateAction.key,                                               
                     ^^^                                                
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:176:26: Error: Too many positional arguments: 0 allowed, but 1 found.
Try removing the extra positional arguments.                            
    return CallbackAction(                                              
                         ^                                              
../../flutter/packages/flutter/lib/src/widgets/actions.dart:348:3: Context: Found this candidate, but the arguments don't match.
  CallbackAction({@required this.onInvoke}) : assert(onInvoke != null); 
  ^^^^^^^^^^^^^^                                                        
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:185:29: Error: 'ActionFactory' isn't a type.
    _actionMap = <LocalKey, ActionFactory>{                             
                            ^^^^^^^^^^^^^                               
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:186:22: Error: Getter not found: 'key'.
      ActivateAction.key: _createAction,                                
                     ^^^                                                
../../flutter/.pub-cache/hosted/pub.dartlang.org/cupertino_list_tile-0.1.0/lib/src/list_tile_background.dart:344:16: Error: The argument type 'Map<LocalKey, invalid-type>' can't be assigned to the parameter type 'Map<Type, Action<Intent>>'.
 - 'Map' is from 'dart:core'.                                           
 - 'LocalKey' is from 'package:flutter/src/foundation/key.dart' ('../../flutter/packages/flutter/lib/src/foundation/key.dart').
 - 'Type' is from 'dart:core'.                                          
 - 'Action' is from 'package:flutter/src/widgets/actions.dart' ('../../flutter/packages/flutter/lib/src/widgets/actions.dart').
 - 'Intent' is from 'package:flutter/src/widgets/actions.dart' ('../../flutter/packages/flutter/lib/src/widgets/actions.dart').
      actions: _actionMap,                                              
               ^            
```

**Comment by jpnurmi** [comment]
Thanks for bringing this up! :+1: I've been using Flutter's master channel with desktop support, so I actually had the changes already prepared, but just kept them in a separate dev branch. I've updated master and published 0.1.1 on pub.dev.

**Comment by MarsGoatz** [comment]
Thank you! Really appreciate your work on this!
